pkgs/applications/networking/cluster/minikube/localkube.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/pkg/minikube/bootstrapper/localkube/localkube.go b/pkg/minikube/bootstrapper/localkube/localkube.go
+index 1c4b5000..c9f120d4 100644
+--- a/pkg/minikube/bootstrapper/localkube/localkube.go
++++ b/pkg/minikube/bootstrapper/localkube/localkube.go
+@@ -113,14 +113,9 @@ func (lk *LocalkubeBootstrapper) UpdateCluster(config bootstrapper.KubernetesCon
+ 
+ 	copyableFiles := []assets.CopyableFile{}
+ 	var localkubeFile assets.CopyableFile
+-	var err error
+ 
+ 	//add url/file/bundled localkube to file list
+-	lCacher := localkubeCacher{config}
+-	localkubeFile, err = lCacher.fetchLocalkubeFromURI()
+-	if err != nil {
+-		return errors.Wrap(err, "Error updating localkube from uri")
+-	}
++	localkubeFile = assets.NewBinDataAsset("out/localkube", "/usr/local/bin", "localkube", "0777")
```

Comment:
How does this work?

Reply:
The old behaviour (we made use) of was that minikube came with a local version of `localkube`.
In 0.23.0 this was changed so that minikube itself is much thinner and auto-downloads `localkube`. This patch reverts to using the local locations again.
Technically, this command looks in three locations and requires one of them to exist. `out/localkube` is what will kick in for us. But I kept the other two locations to stay with the original code.

Reply:
Ah I just reacted negatively to the /usr/local/bin 😄 I'd probably take that out. Feels really weird to be intentionally patching in an impure location (even if unused), especially to someone who doesn't know intimately how the code works (and who runs on a platform without a working sandbox) 😄 feels like someone adjusting the expression in future might accidentally fall through to impure behavior.

Reply:
Makes sense, I'll take it out tomorrow morning

Reply:
Thanks! Otherwise it looks great, and thanks for fixing it!